packages/components/src/alignment-matrix-control/README.md

```diff
@@ -5,14 +5,61 @@ AlignmentMatrixControl components enable adjustments to horizontal and vertical
 ## Usage
 
 ```jsx
-import { AlignmentMatrixControl } from '@wordpress/components';
+import { __experimentalAlignmentMatrixControl as AlignmentMatrixControl } from '@wordpress/components';
 import { useState } from '@wordpress/element';
 
 const Example = () => {
-	const [ alignment, setAlignment ] = useState( 'center center' );
+	const [alignment, setAlignment] = useState('center center');
 
 	return (
-		<AlignmentMatrixControl value={ alignment } onChange={ setAlignment } />
+		<AlignmentMatrixControl
+			value={alignment}
+			onChange={(newAlignment) => setAlignment(newAlignment)}
+		/>
 	);
 };
 ```
+
+## Props
+
+The component accepts the following props:
+### className
+
+The class that will be added with "component-alignment-matrix-control" to the classes of the wrapper <Composite/> component.
+If no className is passed only "component-alignment-matrix-control" is used.
+
+-   Type: `String`
+-   Required: No
+
+### id
+
+Unique ID for the component.
+-  Type: `String`
+-  Required: No
+### label
+
+If provided, sets the aria-label attribute of the wrapper <Composite/> component.
+
+-   Type: `String`
+-   Required: No
+-   Default: `Alignment Matrix Control`
+### defaultValue
+
+If provided, sets the default alignment value.
+- Type: `String`
+- Required: No
+- Default: `center center`
+
+### onChange
+
+A function that receives the updated alignment value.
+
+-   Type: `function`
```

Comment:
Could we replace types to be closer to how TypeScript types are described? 

In this case, it could be something like `( nextValue: string ) => void`


Other changes would be:

- from `String` to `string`
- from `Number` to `number`